CIPs/cip-50.md

```diff
@@ -0,0 +1,50 @@
+---
+cip: 50
+title: Support SM-2/3 verification as internal contracts
+author: Guang Yang (@Regulusyg)
+discussions-to: XXX
```

Comment:
Please add a link for discussion. I suggest you create an issue in this repo for this purpose.